src/main/java/de/tum/in/www1/artemis/service/ProgrammingExerciseService.java

```diff
@@ -918,16 +920,17 @@ public void importBuildPlans(final ProgrammingExercise templateExercise, final P
         // running the plan for the first time
         continuousIntegrationService.get().copyBuildPlan(templateKey, templatePlanName, targetKey, targetName, templatePlanName);
         continuousIntegrationService.get().copyBuildPlan(templateKey, solutionPlanName, targetKey, targetName, solutionPlanName);
+        continuousIntegrationService.get().giveProjectPermissions(targetKey, CIRole.LOGGED_IN, List.of(CIPermission.READ));
```

Comment:
Can we only give the instructors and tutors groups read permissions? I don't think that logged in and anonymous users need to see this.

This is something that we should also do when generating the exercise, so that the Bamboo project is not visible to everyone

Reply:
I give create and read permissions to instructors and only read permissions to tutors now (both for imports and new exercises). 
Since Bamboo always gives read permissions to anonymous and logged in users for new exercises, there now is also a method, which removes all default permissions if there are any.